packages/components/src/components/SquigglePlayground/index.tsx

```diff
@@ -110,20 +110,30 @@ export const SquigglePlayground: React.FC<SquigglePlaygroundProps> = (
   const [project] = useState(() => {
     // not reactive on `linker` changes; TODO?
     return new SqProject({ linker });
+  }); // TODO: Maybe this could go into useSquiggleRunner?
```

Comment:
Yeah, `linker` on standalone `SetupSettings` could be useful.

Reply:
One challenge is that ``SquigglePlayground`` uses ``project``, not ``standalone``. ``project`` only works when the ``project`` is passed in, so it could add in ``linker``, but then it would need to use ``useEffect`` or similar to forcibly update the ``project`` when ``linker`` changes.

Any advice here?

Reply:
- if you switch to `standalone` with linker, `useSquiggleRunner` (or, more exactly, `useSetup`) would be responsible for `linker` -> `project` updates; you would get `project` back from `useSquiggleRunner` so I don't expect any problems there
- linker shouldn't change very often, but when it does, it's reasonable to reset the entire project
- I mostly want `linker` in props for other components, `<SquiggleModelExportPage>` and so on